### DISCUSSION TITLE
Australian Use

### DISCUSSION THREAD

**Discussion by neohusky** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#46] [Q&A]
Hi

I’ve moved into a new home that has a Solar array on the roof.  I had previously considered energy monitoring in my old home using emonPi, but, never got around to it. Having solar with any home energy monitor seems a bit of a missed opportunity.

I’ve setup home assistant and have a few ESP32 devices connected for various other functions. I was wondering about the suitability of the expandable-6-channel-esp32-energy-monitor for my situation (Australian SIngle phase 230V 50Hz supply + Solar array).

I’d like to certainly monitor:
Solar supply
Grid supply
some individual circuit monitoring (eg lights + power circuits)

If this is suitable can you make some recommendations hardware setup I’d need to get started or at least point me in the right direction.

Thanks in advance.
Theo

**Comment by CircuitSetup** [comment]
Hi Theo,

Thanks for considering our metering products!

The 6 channel meter works great for your setup, plus you'll be able to
monitor 4 other circuits. You would need the meter, ESP32, and at least 2
larger current transformers. The 120A (16mm) or 100A (13mm) will probably
fit around your main and solar wires. And the 20A or 80A CTs will fit
around the others. The only other thing you would need is an AC transformer
that brings down the voltage to between 9-12VAC. It should output about
1000mA in current.

Let me know if you have any other questions!

Best,
John

**Reply by neohusky** [reply]
Thank you very much John for taking the time to put together such a detailed reply. 

I look forward to trying it out soon.

**Reply by andrewmov** [reply]
Hi Theo I also live in Australia and am using the 6channel board with solar.  I am still struggling with the setup and cant find much documentation for setup in Australia.  Do you have any advice or recommendations for setup?

Thanks, 
Andrew
